subscriptionform/templates/sports/team_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
```

Comment:
dit voelt als een generieke pagina en volgens mij heb ik die eerder gezien. Je zou dus de template in de view aan kunnen passen naar 1 pagina en die voor meerdere onderdelen kunnen gebruiken...